docs/typescript.md

```diff
@@ -164,7 +164,7 @@ You can explore the syntax more in the [TypeScript playground][tsplay].
 
 - [TypeScript Handbook][ts-handbook]
 - [React's documentation on TypeScript][react-ts]
-- [React + TypeScript Cheatsheets][cheat] has a good overview on how to use React with TypeScript
+- [React + TypeScript Cheatsheets][cheats] has a good overview on how to use React with TypeScript
```

Comment:
Looks like this is a broken link. Could you please remove these from your PR?

Reply:
Remove the entire line ?
-  - [React + TypeScript Cheatsheets][cheat] has a good overview on how to use React with TypeScript